notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -4,7 +4,7 @@
       "cell_type": "code",
```

Comment:
<p>Run the following example request to create your own custom training job using the container you just built and uploaded to Artifact Registry. Then training results are streamed to TensorBoard.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>